### PR TITLE
About us Javascript Dom

### PR DESCRIPTION
**Pull request by JohnnyAstrom**
Finalised version 1.0 of about us DOM

### CODE REVIEW

File: pages/about/about.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+createSections();
+
```
Comment (Nishune) [inline pages/about/about.js:2]: On top here, it seems according to the figma prototype that you have missed adding the <h1> element that says "Om Kino Sandviken". Also the picture is missing that should be appended into and rendered into the HTML.

**Review by Nishune — CHANGES_REQUESTED** [review]
### Code review

Overall the code looks solid, but it seems you have missed the <h1> element which i commented in the code. Also the picture is missing according to the figma prototype. This should be fixed before merge.

File: pages/about/about.js (hunk 1 above)
Comment (JohnnyAstrom) [reply pages/about/about.js:2]: Missed that. Perfect will fix it at once! 👍

Comment (JohnnyAstrom) [reply pages/about/about.js:2]: Made changes so that it has a headline. Planning to make the "om oss picture" as final step in mobile CSS i think.

**Review by Nishune — APPROVED** [review]

**Comment by Nishune** [comment]
Looks good now, also heard your reasoning on voice chat regarding the picture.

**Pull request merged**